src/components/JobCard.js

```diff
@@ -5,7 +5,7 @@ import {Row, Col,Badge} from "react-bootstrap"
 import moment from 'moment'
 import {useHistory} from 'react-router-dom'
 
-export default function JobList(props) {
+export default function JobCard(props) {
```

Comment:
here, you need to document what type of variables are expected on "job". you _could_ use Flow or https://reactjs.org/docs/typechecking-with-proptypes.html, but here, let's just add some simple documentation to the top of our components with complicated props.

/**
Sample prop `job`:
{
   name: 'asdf'
....
}

**/

Reply:
i add that comments and already commit it